File: TextosJapones/src/main/java/org/jisho/textosJapones/model/enums/Conexao.java
package org.jisho.textosJapones.model.enums;

public enum Conexao {
    MANGAEXTRACTOR("MANGA_EXTRACTOR"),
    NOVELEXTRACTOR("NOVEL_EXTRACTOR"),
    TEXTOINGLES("TEXTO_INGLES"),
    DECKSUBTITLE("DECKSUBTITLE"),
    TEXTOJAPONES("TEXTO_JAPONES"),
    FIREBASE("FIREBASE");

    private final String value;

    Conexao(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return this.value;
    }
}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/model/enums/Conexao.kt
package br.com.fenix.processatexto.model.enums


enum class Conexao(private val value: String, val isExternal : Boolean) {
    PROCESSA_TEXTO("PROCESSA_TEXTO", false),
    MANGA_EXTRACTOR("MANGA_EXTRACTOR", false),
    NOVEL_EXTRACTOR("NOVEL_EXTRACTOR", false),
    TEXTO_INGLES("TEXTO_INGLES", false),
    TEXTO_JAPONES("TEXTO_JAPONES", false),
    DECKSUBTITLE("DECKSUBTITLE", false),
    FIREBASE("FIREBASE", true);

    @Override
    override fun toString(): String {
        return value
    }

}